### ISSUE TITLE
refactor inline_var throws 'containing_statement is nil" 

### ISSUE THREAD

**Issue by pintjuk** [ThePrimeagen/refactoring.nvim#456]
Hi this plugin looks amazing and I was excited about trying it out.

## What I did:
Installed the plug-in according to the readme 

in visual mod I place the cursor on a variable in and do ":Refactor inline_var"

## What I expect
the variable gets inlined

## What happens
I see the error: "containing_statement is nil"

the error is thrown here [here](https://github.com/ThePrimeagen/refactoring.nvim/blob/d2786877c91aa409c824f27b4ce8a9f560dda60a/lua/refactoring/tasks/node_on_cursor_setup.lua#L23)

I was trying to figure out what this part of the code does and it looks like it is failing to fetch a referees to the symbol under the cursor for some reason. But I am a bit lost in this code base, so it would be grate if somebody could give me pointers on how to further investigate this. 

Perhaps just forgot to install something, in that case I would be thankful is somebody pointed that out.

**Comment by TheLeoP** [comment]
Could you share a minimal example of the code and language that you were trying to refactor when you found this error?

**Comment by pintjuk** [comment]
it is golang

```go
package main

import "fmt"
var (
	PI = 3.14
)

func main() {
	
	fmt.Println("Hello, World!", PI)
	
}
```

**Comment by TheLeoP** [comment]
It looks like we were only considering the short variable declarations. This should be fixed by #458
